src/components/SlateEditor/plugins/related/EditRelated.jsx

```diff
@@ -1,304 +0,0 @@
-  }
-
-  toggleAddExternal() {
-    this.setState(prevState => ({
-      showAddExternal: !prevState.showAddExternal,
-    }));
-  }
-
-  async searchForArticles(query, page) {
-    return search({
-      query,
-      page,
-      'context-types': 'standard, topic-article',
-    });
-  }
-
-  render() {
-    const {
-      onRemoveClick,
-      updateArticles,
-      insertExternal,
-      articles,
-      onInsertBlock,
-      onExit,
-      t,
-      ...rest
-    } = this.props;
-
-    return (
-      <div>
-        <Overlay onExit={onExit} />
-        <div
-          ref={placeholderEl => {
-            this.placeholderEl = placeholderEl;
-          }}>
-          <Portal isOpened>
-            <StyledBorderDiv
-              ref={embedEl => {
-                this.embedEl = embedEl;
-              }}
-              {...rest}>
-              <h1 className="c-section-heading c-related-articles__component-title">
-                {t('form.related.title')}
-              </h1>
-              <p>{t('form.related.subtitle')}</p>
-              <StyledListWrapper>
-                <DragDropContext
-                  onDragEnd={a => {
-                    if (!a.destination) {
-                      return;
-                    }
-                    const toIndex = a.destination.index;
-                    const fromIndex = a.source.index;
-                    const newArticles = [...articles];
-
-                    var element = newArticles[fromIndex];
-                    newArticles.splice(fromIndex, 1);
-                    newArticles.splice(toIndex, 0, element);
-                    updateArticles(newArticles);
-                  }}>
-                  <Droppable droppableId="relatedArticleDroppable">
-                    {(provided, snapshot) => (
-                      <div
-                        ref={provided.innerRef}
-                        className={snapshot.isDraggingOver ? 'drop-zone dragging' : 'drop-zone'}>
-                        {articles.map((article, index) => {
-                          if (!article) {
-                            return null;
-                          }
-                          const articleKey =
-                            article.id === ARTICLE_EXTERNAL ? article.tempId : article.id;
-                          return (
-                            <Draggable key={articleKey} draggableId={articleKey} index={index}>
-                              {(providedInner, snapshotInner) => (
-                                <div
-                                  className="drag-item"
-                                  ref={providedInner.innerRef}
-                                  {...providedInner.dragHandleProps}
-                                  {...providedInner.draggableProps}>
-                                  <StyledArticle
-                                    isDragging={snapshotInner.isDragging}
-                                    dragHandleProps={providedInner.dragHandleProps}
```

Comment:
Er disse unødvendige?

Reply:
Kan være noe som henger igjen fra en gammel versjon av react-beautiful-dnd, men ser ikke noen referanser til denne typen bruk i dokumentasjonen nå. Typescript klager forøvig på at disse propsene ikke finnes.